master/custom/builders.py

```diff
@@ -125,6 +126,8 @@
     ("aarch64 RHEL8 LTO", "cstratak-RHEL8-aarch64", LTONonDebugUnixBuild),
     ("aarch64 RHEL8 LTO + PGO", "cstratak-RHEL8-aarch64", LTOPGONonDebugBuild),
 
+    ("aarch64 Ubuntu 22.04 BigMem", "diegorusso-aarch64-bigmem", UnixBigmemBuild),
```

Comment:
We need to start in the unstable set, around line 245.  Once we've proven that things work as expected, we can move back here to the stable set.

Reply:
Moved, thanks!